### DISCUSSION TITLE
the project future

### DISCUSSION THREAD

**Discussion by alenat10** [Azure/CanadaPubSecALZ#406] [General]
Hi, I was wondering what the plans for the project future are. It looks like it is not under active development anymore (for example it is not switched to AMA from MMA). There is a parallel active project by MS UK; this one is more feature-rich from my point of view and oriented to my country, but if it will stay frozen in year 2023 state, I am afraid I won't be able to deploy/recommend it to my Customers... 
That is why an information for the project future is important for a right decision on a toolset. We tried to ping MS Canada on it and did not get any reply.

**Comment by Vallentyne** [comment]
Hello!
We have absolutely not been active for a longer while than intended!  The project is open-sourced but of course there are maintainers and owners from inside Microsoft that drive much of the change, myself included.  The reality of course is that this project is not our "day job" directly, but rather adjacent to it.  :-)

I have recently done a bit of maintenance to the GitHub backend and have a few changes I hope to push in the coming weeks that are an effort to sweep out some of the dust.

So, not dead yet, just sleeping!

**Comment by alenat10** [comment]
Thank you for the update and your (and your peers)  hard work, Vallentyne. That is what I was afraid of - reading the article "Microsoft Canada launches ALZ for CPS" (https://www.microsoft.com/en-ca/industry/blog/government/2021/11/16/microsoft-canada-launches-azure-landing-zone-for-canadas-public-sector/) on Microsoft site some of my Customers (and Consutants here in Canada) may mistakenly consider this is a product backed up by Microsoft (including regular updates and support), while in reality it is an open source project supported by busy enthusiasts in their very limited free time. That is really pity because another "Microsoft" ALZ project developed by folks from Microsoft UK is much more alive but has no Canadian specific at all, of course. I like your approach and level of automation (especially for App Landing Zones) better, but unfortunately since the project went into a sleep mode just before Microsoft deprecated MMA-based monitoring switching to AMA it is hard to use now.
